basic_text_model_WIP.py

```diff
@@ -0,0 +1,89 @@
+import pandas as pd
+from sklearn.pipeline import Pipeline
+from sklearn.feature_extraction.text import CountVectorizer
+from sklearn.linear_model import LogisticRegression
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.metrics import classification_report, ConfusionMatrixDisplay
+from pathlib import Path
+from preprocess_data import split_train_val_test
+import matplotlib.pyplot as plt
+import numpy as np
+
+czech_stop_list = [
```

Comment:
can be reused later by other modules, make it a separate importable file?

If this is constant, I suggest to use upper casing: `CZECH_STOPWORDS = `